src/errors/types.rs

```diff
@@ -502,6 +503,7 @@ impl ErrorType {
             Self::IntParsingSize => "Unable to parse input string as an integer, exceeded maximum size",
             Self::FloatType => "Input should be a valid number",
             Self::FloatParsing => "Input should be a valid number, unable to parse string as an number",
```

Comment:
```suggestion
            Self::FloatParsing => "Input should be a valid number, unable to parse string as a number",
```
I know this typo isn't a change in your PR but would appreciate it if we could fix it in this PR anyway